src/utils/send_tx.ts

```diff
@@ -94,9 +95,18 @@ export async function getComputeBudgetInstructions(
 
     const data = await response.json();
     if (data.error) {
-      throw new Error("Error fetching priority fee from Helius API");
```

Comment:
I see no reason why a send transaction should fail because of a failed helius api call to get priority fees @slimeonmyhead

Reply:
Maybe this should be a PR to the original repo instead